src/MultiReport/SystemControl.cpp

```diff
@@ -82,8 +78,10 @@ void SystemControl_::press(uint8_t s) {
         USBDevice.wakeupHost();
 #endif
 #ifdef ARDUINO_ARCH_SAMD
-        extern USBDevice_SAMD21G18x usbd;
-        usbd.wakeupHost();
+        // This is USBDevice_SAMD21G18x.wakeupHost(). But we can't include that
+        // header, because it redefines a few symbols, and causes linking
+        // errors. So we simply reimplement the same thing here.
+        USB->DEVICE.CTRLB.bit.UPRSM = 1;
```

Comment:
Do ALL SAMD devices use CTRLB there?

Reply:
All that are supported by `ArduinoCore-SAMD`, yes. Mind you, this alone doesn't do the trick, and I haven't figured out how to make wakeup work with SAMD yet.

Reply:
ok